osgp/platform/osgp-domain-core/src/main/java/org/opensmartgridplatform/domain/core/valueobjects/FirmwareModuleData.java

```diff
@@ -56,23 +56,30 @@
      * FirmwareFile should equal {@link #moduleVersionMBusDriverActive}.
      */
     public static final String MODULE_DESCRIPTION_MBUS_DRIVER_ACTIVE = "m_bus_driver_active_firmware";
+    /**
+     * Description of the FirmwareModule for which the module version in a
+     * FirmwareFile should equal {@link #moduleVersionSimple}.
+     */
+    public static final String MODULE_DESCRIPTION_SIMPLE_VERSION_INFO = "simple_version_info";
 
     private final String moduleVersionComm;
     private final String moduleVersionFunc;
     private final String moduleVersionMa;
     private final String moduleVersionMbus;
     private final String moduleVersionSec;
     private final String moduleVersionMBusDriverActive;
+    private final String moduleVersionSimple;
 
     public FirmwareModuleData(final String moduleVersionComm, final String moduleVersionFunc,
             final String moduleVersionMa, final String moduleVersionMbus, final String moduleVersionSec,
-            final String moduleVersionMBusDriverActive) {
+            final String moduleVersionMBusDriverActive, final String moduleVersionSimple) {
         this.moduleVersionComm = moduleVersionComm;
         this.moduleVersionFunc = moduleVersionFunc;
         this.moduleVersionMa = moduleVersionMa;
         this.moduleVersionMbus = moduleVersionMbus;
         this.moduleVersionSec = moduleVersionSec;
         this.moduleVersionMBusDriverActive = moduleVersionMBusDriverActive;
+        this.moduleVersionSimple = moduleVersionSimple;
```

Comment:
The other versions were included in the `toString` representaton of the FirmwareModuleData. Probably best to include the simple version info there as well.

Reply:
done